### ISSUE TITLE
Font-size on the front page is too small

### ISSUE THREAD

**Issue by padenot** [mdn/MDN-feedback#13]
16px is generally recommended size, and it's what other MDN pages that I just looked at have.

It's 13px on the front page, and it's too small to be legible.

**Comment by estelle** [comment]
Agree. Inspector showed 10.888px in areas on the home page. 16 would be better, but minimally 14. Default line height of 1.4 instead of 1.2 would also help. 

Do love the new design. Great job folks!

**Comment by schalkneethling** [comment]
I think we had at as `100%/1.7` in the past. I wonder if we should consider `100%/1.5`?

**Comment by estelle** [comment]
> I think we had at as `100%/1.7` in the past. I wonder if we should consider `100%/1.5`?

On the home page,line height is set at 1.2 on the body. 1.5 and 1.7 would both be better, but at least 1.4 please.
